src/main/java/org/jabref/gui/entryeditor/EntryEditor.java

```diff
@@ -411,4 +411,20 @@ public void updatePreviewInTabs(PreviewPreferences previewPreferences) {
             }
         }
     }
+
+    public void nextPreviewStyle() {
+        for (Tab tab : this.entryEditorTabs) {
+            if (tab instanceof FieldsEditorTab) {
+                ((FieldsEditorTab) tab).previewPanel.nextPreviewStyle();
```

Comment:
I would add `next/previousPreviewStyle` to `EntryEditorTab` and override it properly in `FieldsEditorTab` . In this way, we are a bit more flexible in the future (e.g. if there are other tabs with previews embedded) and you can make `previewPanel` private (at least it should be private).

Reply:
Done.